app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,704 @@
+ <resources>
```

Comment:
@esodigoyo this Spanish translation was recently emailed to me by someone named Roberto Espiño -- I'm letting you know because you had offered to translate the app into Spanish and Greek in [this comment](https://github.com/atorch/probability_puzzles/issues/5#issuecomment-1212166456)